src/main/java/org/jenkinsci/test/acceptance/utils/SystemEnvironmentVariables.java

```diff
@@ -0,0 +1,28 @@
+package org.jenkinsci.test.acceptance.utils;
+
+/**
+ * Gets a system environment variable. First it checks if the variable is an environment variable and if it is the case,
+ * it returns the configured value.
+ * If not then it checks if it is defined as java system property, and if not it returns the default value.
+ */
+public class SystemEnvironmentVariables {
+
+    public static final String getPropertyVariableOrEnvironment(String property, String def) {
```

Comment:
As @olivergondza said I think this is not something in this PR but global strategy.